eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -100,6 +108,17 @@ data GovEnv era = GovEnv
   , gePParams :: !(PParams era)
   }
 
+data GovRuleState era = GovRuleState
+  { grsGovSnapshots :: GovSnapshots era
+  , grsVState :: VState era
```

Comment:
Are we now storing `VState` in two places?

Reply:
ah, I suppose it's an ephemeral state

Reply:
Yes, it is just yet another state-change, just like another other rule.